packages/core/src/endowmentsToolkit.js

```diff
@@ -525,7 +525,7 @@ function endowmentsToolkit({
   function copyWrappedGlobals(
     globalRef,
     target,
-    globalThisRefs = ['globalThis']
+    globalThisRefs = ['window', 'self', 'global', 'globalThis', 'top', 'frames', 'parent']
```

Comment:
(must) do not change this default. we are supposed to pass these in if we want more than just `globalThis`

Reply:
11f8e45